src/aks-preview/azext_aks_preview/tests/latest/data/httpproxyconfig.json

```diff
@@ -4,6 +4,5 @@
     "noProxy": [
         "localhost",
         "127.0.0.1"
-    ],
-    "trustedCa": "G9yR2xrQ29NRjNURHg4cm1wOURCaUIvCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
```

Comment:
I confirmed with Node Provisioning team, currently HttpProxy config with trustedCA doesn't work. So generate this recording test without trustedCA.